utils/prometheus/Cargo.toml

```diff
@@ -10,11 +10,15 @@ repository = "https://github.com/paritytech/substrate/"
 
 [dependencies]
 log = "0.4.8"
-prometheus = "0.7"
 futures-util = { version = "0.3.1", default-features = false, features = ["io"] }
 derive_more = "0.99"
 
+# WASM
+[target.'cfg(target_os = "unknown")'.dependencies]
+prometheus = "0.7"
+
 [target.'cfg(not(target_os = "unknown"))'.dependencies]
 async-std = { version = "1.0.1", features = ["unstable"] }
 hyper = { version = "0.13.1", default-features = false, features = ["stream"] }
+prometheus = { version = "0.7", features = ["process"] }
```

Comment:
Cargo merges features. So, this will break the browser build.

Reply:
Hmm, that's odd. It seems to compile fine for me.

Reply:
Maybe [this](https://github.com/tikv/rust-prometheus/blob/6949f4d7cd0d7461839acfc62bf809a8fa14b71c/src/lib.rs#L152) safes us.

However, it looks like this could break windows.^^